src/main/java/frc/robot/subsystems/EverybotIntakeSubsystem.java

```diff
@@ -0,0 +1,65 @@
+
+// Copyright (c) FIRST and other WPILib contributors.
+// Open Source Software; you can modify and/or share it under the terms of
+// the WPILib BSD license file in the root directory of this project.
+package frc.robot.subsystems;
+
+
+import com.revrobotics.CANSparkMax;
+
+
+import edu.wpi.first.wpilibj2.command.SubsystemBase;
+import frc.robot.hardware.MotorController;
+import frc.robot.hardware.MotorController.MotorConfig;
+import edu.wpi.first.networktables.GenericEntry;
+import edu.wpi.first.wpilibj.shuffleboard.Shuffleboard;
+import edu.wpi.first.wpilibj.shuffleboard.ShuffleboardTab;
+
+public class EverybotIntakeSubsystem extends SubsystemBase implements AutoCloseable {
+  public static final double kOuttakeSpeed = -1;
+  public static final double kIntakeSpeed = 0.5;
+  private CANSparkMax motor;
+  private static ShuffleboardTab intakeTab = Shuffleboard.getTab("Match");
+  private static GenericEntry intakeEntry = intakeTab.add("Intake Speed", 0.0).getEntry();
```

Comment:
The duplicate entry in IntakeSubsystem is breaking the JUnit tests.
```suggestion
  private static GenericEntry intakeEntry = intakeTab.add("EverybotIntake Speed", 0.0).getEntry();
```